.gitignore

```diff
@@ -5,3 +5,5 @@ log
 *.pid
 npm-debug.log
 config/index.js
+.vscode
+cordlr.json
```

Comment:
It's not needed in here, because the cordlr.json should never be in the cordlr-cli folder.
Just remove that line, it's fine! :)